Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -244,14 +244,42 @@ theorem stabilizer_smul_eq_stabilizer_map_conj (g : G) (a : α) :
   rw [mem_stabilizer_iff, ← smul_left_cancel_iff g⁻¹, smul_smul, smul_smul, smul_smul,
     inv_mul_cancel, one_smul, ← mem_stabilizer_iff, Subgroup.mem_map_equiv, MulAut.conj_symm_apply]
 
+variable (g h : G) (a : α)
+
+/-- The natural group equivalence between the stabilizers of two elements in the same orbit. -/
+def stabilizerEquivStabilizer : stabilizer G a ≃* stabilizer G (g • a) :=
+  ((MulAut.conj g).subgroupMap (stabilizer G a)).trans
+    (MulEquiv.subgroupCongr (stabilizer_smul_eq_stabilizer_map_conj g a).symm)
+
+variable {a} in
+theorem stabilizerEquivStabilizer_apply (x : stabilizer G a) :
+    stabilizerEquivStabilizer g a x = MulAut.conj g x := by
+  simp [stabilizerEquivStabilizer]
+
+variable {a} in
+theorem stabilizerEquivStabilizer_symm_apply (x : stabilizer G (g • a)) :
+    (stabilizerEquivStabilizer g a).symm x = MulAut.conj g⁻¹ x := by
+  simp [stabilizerEquivStabilizer]
+
+theorem stabilizerEquivStabilizer_trans :
+    (stabilizerEquivStabilizer g a).trans (stabilizerEquivStabilizer h (g • a)) =
+      (stabilizerEquivStabilizer (h * g) a).trans (MulEquiv.subgroupCongr (by rw [mul_smul])) := by
+  ext x; simp [stabilizerEquivStabilizer]
```

Comment:
Can you also add `stabilizerEquivStabilizer_mul`, which is the flipped version, but with the `subgroupCongr` remaining on the RHS?